File: dotnet/src/Downstairs.Infrastructure/Persistence/Models/Task.cs
using System;
using System.Collections.Generic;

namespace Downstairs.Infrastructure.Persistence.Models;

public partial class Task
{
    public long Id { get; set; }

    public long CustomTaskId { get; set; }

    public long ScheduleEmployeeId { get; set; }

    public string Name { get; set; } = null!;

    public string? Description { get; set; }

    public bool IsCompleted { get; set; }

    public virtual CustomTask CustomTask { get; set; } = null!;

    public virtual ScheduleEmployee ScheduleEmployee { get; set; } = null!;
}

File: php-laraval-app/app/Models/Task.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class Task extends Model
{
    use HasFactory;

    public $timestamps = false;

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'custom_task_id',
        'schedule_employee_id',
        'name',
        'description',
        'is_completed',
    ];

    /**
     * The attributes that should be cast to native types.
     *
     * @var array
     */
    protected $casts = [
        'is_completed' => 'boolean',
    ];

    /**
     * Define the alias of the columns or relations.
     *
     * @var array<string,string>
     */
    protected array $aliases = [
        'schedule_id' => 'schedule_employee_id',
    ];

    public function schedule(): BelongsTo
    {
        return $this->belongsTo(ScheduleEmployee::class, 'schedule_employee_id')->withTrashed();
    }

    public function customTask(): BelongsTo
    {
        return $this->belongsTo(CustomTask::class, 'custom_task_id');
    }
}
